src/usecase/client_user_ucase.go

```diff
@@ -98,10 +99,24 @@ func (cu *ClientUser) Add(c *gin.Context) { //nolint:dupl
 		return
 	}
 
+	// Checking for empty var, then check status and add user if status "deleted"
 	if existingUser.ID != uuid.Nil {
+		if userStatus == "deleted" {
+			user, _ := userRepo.Add(user)
```

Comment:
why you're not handling errors here? user, err := ...
if err != nil {
 ...do something
}